<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameConfigurationDialog</class>
 <widget class="QDialog" name="GameConfigurationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>231</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>206</width>
     <height>354</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(255, 147, 0);
font: 14pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="text">
       <string>Tic-Tac-Toe Game</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="lblPlayer1">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(4, 51, 255);
font: 14pt &quot;Times New Roman&quot;</string>
        </property>
        <property name="text">
         <string>Player 1:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtPlayer1">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="lblPlayer2">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(255, 38, 0);
font: 14pt &quot;Times New Roman&quot;;</string>
        </property>
        <property name="text">
         <string>Player 2:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtPlayer2">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 249, 0);
font: 14pt &quot;Times New Roman&quot;;</string>
        </property>
        <property name="text">
         <string>Game Side:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox"/>
      </item>
      <item>
       <widget class="QSlider" name="horizontalSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Game Mode:</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="radioTwoPlayerMode">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(148, 33, 146);</string>
         </property>
         <property name="text">
          <string>Two-Player Mode</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioAIMode">
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 64, 255);
background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>AI Mode</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GameConfigurationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>214</x>
     <y>363</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GameConfigurationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>214</x>
     <y>363</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtPlayer1</sender>
   <signal>textChanged(QString)</signal>
   <receiver>GameConfigurationDialog</receiver>
   <slot>updateOkBtn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>155</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtPlayer2</sender>
   <signal>textChanged(QString)</signal>
   <receiver>GameConfigurationDialog</receiver>
   <slot>updateOkBtn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>102</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>171</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>113</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updateOkBtn()</slot>
 </slots>
</ui>
